src/odemis/driver/smaract.py

```diff
@@ -2549,6 +2549,17 @@ def __init__(self, name, role, locator, ref_on_init=False, axes=None, speed=1e-3
             self._set_accel(channel, accel)
             self._set_hold_time(channel, hold_time)
 
+            try:
+                # Log referencing mode, and warn if it's not normal
+                ref_mode = self.GetProperty_i32(SA_CTLDLL.SA_CTL_PKEY_POSITIONER_TYPE, channel)
+                log_lvl = logging.INFO
+                if ref_mode != SA_CTLDLL.SA_CTL_REF_OPT_BIT_NORMAL:
```

Comment:
BIT_NORMAL is a shortcut for 0. However, there are some bits which are completely fine. The one to pay attention to is `SA_CTL_REF_OPT_BIT_AUTO_ZERO`.
=> `if ref_mode & SA_CTLDLL.SA_CTL_REF_OPT_BIT_AUTO_ZERO:`